docs/api/navigators/StackNavigator.md

```diff
@@ -137,7 +137,7 @@ Color for material ripple (Android >= 5.0 only)
 
 #### `gesturesEnabled`
 
-Whether you can use gestures to dismiss this screen. Defaults to true on iOS, false on Android
+Whether you can use gestures to dismiss this screen. Defaults to true on iOS, false on Android. This value when set is only respected in `mode` == `card`.
```

Comment:
@ericvicenti in beta 8 neither your approach or @nihgwu's one are not working, the gestures are not disabled in both cases. Does this PR solves this issue?